website/src/roadmap/portability.md

```diff
@@ -151,13 +151,33 @@ for details.
 
 ### Running Python wordcount on Flink {#python-on-flink}
 
+<<<<<<< HEAD
```

Comment:
Please resolve this (I removed the runner-specific instructions from this page because I thought runner-specific instructions were better placed in their own respective documentation).